src/dict.c

```diff
@@ -1432,9 +1432,11 @@ static int _dictShrinkIfNeeded(dict *d) {
     /* Incremental rehashing already in progress. Return. */
     if (dictIsRehashing(d)) return DICT_OK;
     
-    if (dict_can_resize == DICT_RESIZE_ENABLE && 
-        DICTHT_SIZE(d->ht_size_exp[0]) > DICT_HT_INITIAL_SIZE && 
-        (d->ht_used[0] * 100 / DICTHT_SIZE(d->ht_size_exp[0]) < dict_force_shrink_ratio)) 
+    if (DICTHT_SIZE(d->ht_size_exp[0]) > DICT_HT_INITIAL_SIZE && 
```

Comment:
maybe use an early exit for the first condition, to make the later more readable?